_posts/2019-04-25-mar-updates.md

```diff
@@ -0,0 +1,42 @@
+---
+title: March Updates -  A new Kubernetes operator & more Cloud Native Apps. 
+author: Hong Phuc Dang 
+date: 2019-04-25
+banner:
+  image: mar-banner.jpg
+---
+### This is a recap of open source activities and development at Zalando in the month of March
+
+## Project Highlights 
+
+A new operator is added to Zalando’s list of Cloud Native Applications.  [Elasticsearch Operator](https://github.com/zalando-incubator/es-operator) - an operator for running Elasticsearch in Kubernetes with focus on operational aspects, like safe draining and offering auto-scaling capabilities for Elasticsearch data nodes, rather than just abstracting manifest definitions.
+
+To make things even simpler for developers, we also released a new framework that helps to build Kubernetes operators in Python. [Kopf - Kubernetes Operator Pythonic Framework](https://github.com/zalando-incubator/kopf) - a  framework and a library to make Kubernetes operators development easier, just in few lines of Python code. The main goal is to bring the Domain-Driven Design to the infrastructure level, with Kubernetes being an orchestrator/database of the domain objects (custom resources), and the operators containing the domain logic (with no or minimal infrastructure logic).
+
+**Weekly Open Source Maintainer Day at Developer Productivity** - Pionioneer in open source development at Zalando, Platform Infrastructure department has actively released free software and contributed to many upstream projects. Since the beginning of this year, the Cloud Infrastructure team leading by [Jannis Rake-Revelant](https://twitter.com/jannis_r), reserved every Tuesday for open source maintaining work - which they called ‘Maintainer Tuesday’. It is a part of Zalando commitment to open source community. 
```

Comment:
"Pioneering"

Reply:
"team led by"

Reply:
"It is part of Zalando's commitment to open source"

Reply:
@hjacobs fixed